java/ql/src/Security/CWE/CWE-798/SensitiveApi.qll

```diff
@@ -202,7 +203,8 @@ private predicate javaApiCallableUsernameParam(string s) {
   s = "sun.tools.jconsole.ProxyClient;getConnectionName(String, String);1" or
   s = "sun.tools.jconsole.ProxyClient;getProxyClient(String, int, String, String);2" or
   s = "sun.tools.jconsole.ProxyClient;getConnectionName(String, int, String);2" or
-  s = "com.amazonaws.auth.BasicAWSCredentials;BasicAWSCredentials(String, String);0"
+  s = "com.amazonaws.auth.BasicAWSCredentials;BasicAWSCredentials(String, String);0" or
```

Comment:
The comment above suggests this list is Auto-generated. @smowton will this new addition get overriden if placed here?

Reply:
@smowton  and @pwntester - shall we move these APIs of Azure together with those of AWS to the end of
https://github.com/github/codeql/blob/4d014717b6a109d61ae3f9c803092f8df27ede07/java/ql/src/Security/CWE/CWE-798/SensitiveApi.qll#L509-L516?

Reply:
Yes that looks like a good idea, don't modify any of the predicates flagged autogen'd. They all get OR'd together anyhow:

```
        javaApiCallableUsernameParam(call.getCallee(), i) or
        javaApiCallablePasswordParam(call.getCallee(), i) or
        javaApiCallableCryptoKeyParam(call.getCallee(), i) or
        otherApiCallableCredentialParam(call.getCallee(), i)
```

Reply:
Thanks @smowton for the confirmation. I've made the change.